packages/wallet/src/redux/sagas/message-handler.ts

```diff
@@ -35,9 +38,32 @@ function* handleMessage(payload: RequestObject) {
       break;
     case "CreateChannel":
       yield handleCreateChannelMessage(payload);
+
+      break;
+    case "UpdateChannel":
+      yield handleUpdateChannelMessage(payload);
   }
 }
 
+function* handleUpdateChannelMessage(payload: RequestObject) {
+  const {channelId} = payload.params as any;
+
+  // TODO: Error handling, what if the channel does not exist?
```

Comment:
I think we can add a check for this and return an appropriate `Channel NotFound` error